website/docusaurus.config.js

```diff
@@ -48,6 +48,11 @@ var siteSettings = {
   onBrokenMarkdownLinks: "throw",
   trailingSlash: false,
   themeConfig: {
+    docs:{
+      sidebar: {
+        hideable: true;
+      },
+    },
```

Comment:
There is a `;` after `true` which is causing the build to fail. This could be removed, or replaced with a comma and the build should go through successfully!

`hideable: true`

Reply:
i am so dumb 🙈 - thank you @JKarlavige and i've changed this to a comma. looks like it's passing now 🙏